src/main/resources/mturk/worker-ui/mturk.js

```diff
@@ -31,14 +31,14 @@ function initMturk(platformName, workerServiceUrl, experiment) {
         preview: preview
     });
     //load initial bits
-    CreativeCrowd.load();
-    CreativeCrowd.onSubmitAnswer(function (viewData, submittedData) {
+    WorkerUI.load();
```

Comment:
WorkerUI.load() should be called after all initialization (e.g. setting callbacks) is done, because they might not be properly registered then